### DISCUSSION TITLE
How to customize an element to monitor the value of an element in the form and display it in real time

### DISCUSSION THREAD

**Discussion by LingXiongDev** [vueform/vueform#343] [Questions]
Please tell me how to implement an element component to monitor the value of an element in the form and then display the value (the scenario is that the data filled in the first step of the step component is displayed in the next step)
<img width="1439" alt="截屏2024-10-09 下午5 27 53" src="https://github.com/user-attachments/assets/e9e05d95-a31d-481c-862f-438ccc2a9532">

**Comment by adamberecz** [comment]
In general you can reach any element via `form$.el$('path')`. With that you can have access to it's value real-time.

Eg:

```js
element.initWatcher(() => {
  console.log(element.form$.el$('other_el_path').value)
})
```
